bot/seasons/evergreen/movie.py

```diff
@@ -0,0 +1,211 @@
+        if amount > 20:
+            await ctx.send("You can't get more than 20 movies at once. (TMDB limits)")
+            return
+        elif amount < 1:
+            await ctx.send("You can't get less than 1 movies. Just logic.")
+            return
+
+        # Capitalize genre for getting data from Enum, get random page
+        genre = genre.capitalize()
+        page = random.randint(1, 200)
+
+        # Get movies list from TMDB, check is results key in result. When not, raise error. When genre not exist,
+        # show help.
+        try:
+            movies = await self.get_movies_list(self.http_session, MovieGenres[genre].value, page)
+        except KeyError:
+            await ctx.send_help('movies')
+            return
+        if 'results' not in movies.keys():
+            err_text = f'There was problem while fetching movies list. Problematic response:\n```{movies}```'
+            err = Embed(title=':no_entry: Error :no_entry:', description=err_text)
+
+            await ctx.send(embed=err)
+            logger.warning(err_text)
+
+            return
+
+        # Get all pages and embed
+        pages = await self.get_pages(self.http_session, movies, amount)
+        embed = await self.get_embed(genre)
+
+        # Send movies, paginate
+        await ImagePaginator.paginate(pages, ctx, embed)
+
+    async def get_movies_list(self, client: ClientSession, genre_id: str, page: int) -> Dict:
+        """Return JSON of TMDB discover request."""
+        # Define params of request
+        params = {
+            "api_key": TMDB_API_KEY,
+            "language": "en-US",
+            "sort_by": "popularity.desc",
+            "include_adult": "false",
+            "include_video": "false",
+            "page": page,
+            "with_genres": genre_id
+        }
+
+        # Create URL
+        url = BASE_URL + "discover/movie?" + urlencode(params)
+
+        # Make discover request to TMDB
+        async with client.get(url) as resp:
+            data = await resp.json()
+
+        # Return response result
+        return data
+
+    async def get_pages(self, client: ClientSession, movies: Dict, amount: int) -> (List[Tuple[str, str]]):
+        """Fetch all movie pages from movies dictionary. Return list of pages."""
+        pages = []
+
+        for i in range(amount):
+            # Get movie ID, fetch movie information
+            movie_id = movies['results'][i]['id']
+            movie = await self.get_movie(client, movie_id)
+
+            # Create page, append it to pages
+            page, img = await self.create_page(movie)
+            pages.append((page, img))
+
+        return pages
+
+    async def get_movie(self, client: ClientSession, movie: int) -> Dict:
+        """Get Movie by movie ID from TMDB. Return result dictionary."""
+        # Define URL params, generate URL
+        params = {
+            "api_key": TMDB_API_KEY,
+            "language": "en-US"
+        }
```

Comment:
This can be moved to a constant so it doesn't need to be regenerated every time the coro is invoked.